go/dpagg/mean.go

```diff
@@ -66,6 +66,9 @@ type BoundedMeanFloat64 struct {
 	resultReturned bool // whether the result has already been returned
 }
 
+// numSplits sets the number of splits used in bruteforce for Confidence intervals.
+var numSplits = 1000
```

Comment:
We could probably inline this: `var numSplits = 1000 // number of splits used in bruteforce for Confidence intervals`